source/MaterialXView/Viewer.h

```diff
@@ -136,6 +137,11 @@ class Viewer : public ng::Screen
     bool _translationActive;
     ng::Vector2i _translationStart;
 
+    // TextureBaker information
+    mx::TextureBaker _textureBaker;
```

Comment:
Since you've already built the infrastructure to create and store a TextureBakerPtr, I would use that data type here instead of the raw object instance.